### PR TITLE
LUN-432: Change console log linting rule to error

### PR DESCRIPTION
**Pull request by charludo**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/components/AudioPlayer.tsx [hunk 1]
```diff
@@ -54,8 +54,10 @@ const AudioPlayer = ({ document, disabled, submittedAlternative }: AudioPlayerPr
         }
       })
       .catch(async (error: TtsError) => {
+        /* eslint-disable-next-line no-console */
```
Comment (f1sh1918) [inline src/components/AudioPlayer.tsx:57]: is there a reason why not to use `reportError` fkt here?

File: src/components/AudioPlayer.tsx [hunk 2]
```diff
@@ -54,8 +54,10 @@ const AudioPlayer = ({ document, disabled, submittedAlternative }: AudioPlayerPr
...
         console.error(`Tts-Error: ${error.code}`)
         if (error.code === 'no_engine') {
+          /* eslint-disable-next-line no-console */
           await Tts.requestInstallEngine().catch(e => console.error('Failed to install tts engine: ', e))
```
Comment (f1sh1918) [inline src/components/AudioPlayer.tsx:61]: same as above

File: src/services/AsyncStorage.ts [hunk 3]
```diff
@@ -172,7 +172,6 @@ const editUserDocument = async (oldUserDocument: Document, newUserDocument: Docu
 }
 
 const deleteUserDocument = async (userDocument: Document): Promise<void> => {
-  console.log('hi')
```
Comment (steffenkleinle) [inline src/services/AsyncStorage.ts:175]: :smile:

**Review by steffenkleinle — APPROVED** [review]

File: src/components/AudioPlayer.tsx (hunk 1 above)
Comment (charludo) [reply src/components/AudioPlayer.tsx:57]: Wasn't sure this needed reporting. At least on my devices, this happens constantly. Apparently the original implementer also didn't see the value in logging the actual error, so I'm assuming this is only being logged to satisfy `typescript-eslint/no-floating-promises`

**Review by SteffiStuffel — APPROVED** [review]
Looks good :tada:

**Pull request merged**